examples/client/client.c

```diff
@@ -2414,9 +2414,12 @@ THREAD_RETURN WOLFSSL_THREAD client_test(void* args)
         err_sys("unable to load static memory");
     }
 #else
-    ctx = wolfSSL_CTX_new(method(NULL));
-    if (ctx == NULL)
-        err_sys("unable to get ctx");
+    else {
```

Comment:
Instead of the `else` can we just do a check for `if (method)` here instead? With the static memory block above its confusing and prone to possible issues in the future.